pkgs/by-name/me/mesonlsp/package.nix

```diff
@@ -0,0 +1,144 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+
+, gtest
+, makeWrapper
+, meson
+, ninja
+, pkg-config
+, python3
+
+, curl
+, libarchive
+, libpkgconf
+, libuuid
+, nlohmann_json
+
+, git
+, mercurial
+, subversion
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mesonlsp";
+  version = "4.1.3";
+
+  src = fetchFromGitHub {
+    owner = "JCWasmx86";
+    repo = "mesonlsp";
+    rev = "v${version}";
+    hash = "sha256-xniEDRA+YPy9UcIWs8bXkpLsn9NCBEuhy8LrtP5yCTU=";
+  };
+
+  nativeBuildInputs = [
+    gtest
+    makeWrapper
+    meson
+    ninja
+    pkg-config
+    python3
+  ];
+
+  buildInputs = [
+    curl
+    libarchive
+    libpkgconf
+    libuuid
+    nlohmann_json
+  ];
+
+  nativeRuntimeInputs = lib.makeBinPath [
+    git
+    mercurial
+    pkg-config
+    subversion
+  ];
+
+  mesonFlags = [
+    "-Dbenchmarks=false"
+    "-Dtests=false"
```

Comment:
Why the need to disable tests?

Reply:
Well, because there was hardly any need for me to bother about it.